src/Form/ZoomSyncSettingsForm.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\openy_gc_zoom_sync\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * ZoomSync settings form for configure variables.
+ */
+class ZoomSyncSettingsForm extends ConfigFormBase {
+  /**
+   * @return string
+   */
+  public function getFormId() {
+    return 'zoom_sync_setting_form';
+  }
+
+  /**
+   * @return string[]
+   */
+  protected function getEditableConfigNames() {
+    return ['openy_gc_zoom_sync.settings'];
+  }
+
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('openy_gc_zoom_syncs.settings');
+
+    $form['help'] = [
+      '#type' => 'markup',
+      '#markup' => $this->t('This configuration form provides settings of Zoom meetings integration.'),
```

Comment:
Can we have a link to Zoom docs here in description where to get URL and Token for Data Service URL?